src/layman/common/micka/util.py

```diff
@@ -261,7 +240,6 @@ def is_record_exists_exception(root_el):
         root_el[0][0].text.startswith('Record exists')
 
 
-@error_handling_decorator
 def base_insert(xml_str):
```

Comment:
Move base_insert, csw_update, csw_delete to standalone module requests.py or something like that